### PR TITLE
chore(changelog): CHANGELOG.md for v1.6.5 release

### PR DESCRIPTION
**Pull request by docmerlin**
Update CHANGELOG.md for 1.6.5 release

### CODE REVIEW

**Review by samhld — COMMENTED** [review]
@docmerlin I assume the rand function needs docs addition/s? Can you elaborate a little on what "newer InfluxQL" is?

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +25,7 @@ CHANGELOG for Kapacitor master commit 95dd289843c5a556977e129385d27bd511f6dec2
 
 1. [d4fcc6b](https://github.com/influxdata/kapacitor/commit/d4fcc6b): Flux limits and bad flux httpclient
 1. [1959476](https://github.com/influxdata/kapacitor/commit/1959476): Update sarama for  https://github.com/Shopify/sarama/issues/2129
-1. [95dd289](https://github.com/influxdata/kapacitor/commit/95dd289): properly depreciate TLS<1.2, DES and RC4 based ciphers
```
Comment (docmerlin) [inline CHANGELOG.md:14]:
```suggestion
1. [95dd289](https://github.com/influxdata/kapacitor/commit/95dd289): properly depreciate TLS<1.2, DES and RC4 based ciphers
```

**Comment by docmerlin** [comment]
@samhld Yes, it does.  Also newer influxQL was updating influxQL to a newer version that allows statements like subqueries etc.  The version of influxql that kappa was using was ancient.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,7 +25,7 @@ CHANGELOG for Kapacitor master commit 95dd289843c5a556977e129385d27bd511f6dec2
...
+
```
Comment (docmerlin) [inline CHANGELOG.md:28]:
```suggestion
1. [95dd289](https://github.com/influxdata/kapacitor/commit/95dd289): properly depreciate TLS<1.2, DES and RC4 based ciphers
```

**Review by samhld — APPROVED** [review]
LGTM

**Pull request merged**